src/MinFraud/Validation/Rules/Email.php

```diff
@@ -14,7 +14,7 @@ public function __construct()
     {
         $this->validatable = v::keySet(
             v::key('address', v::oneOf(new Md5(), v::email()), false),
-            v::key('domain', v::domain(), false)
+            v::key('domain', v::AlwaysValid(), false)
```

Comment:
We still want to validate that the domain is well formed, we just don't want to do TLD validation.